features/step_definitions/smartanswers_steps.rb

```diff
@@ -10,7 +10,7 @@
 Then /^the slug should be (valid|invalid)$/ do |valid_or_invalid|
   is_valid = valid_or_invalid == "valid"
 
-  error_message = "#current-error"
+  error_message = "#govuk-error-message"
```

Comment:
I think you want this to be a class rather than an ID? `.govuk-error-message`

Reply:
Yes... good point! I've amended the commit.